app/controllers/posts_controller.rb

```diff
@@ -103,13 +103,13 @@ def hidden
 
   def unhide
     if params[:unhide_boards].present?
-      board_ids = params[:unhide_boards].map(&:to_i).compact.uniq
+      board_ids = params[:unhide_boards].filter_map(&:to_i).uniq
```

Comment:
Interesting - I think `.map(&:to_i).compact` wasn't doing anything beyond `.map(&:to_i)` before – see this example, where to_i never seems to return nil:

```rb
irb(main):006:0> ['1', '2', '', 'hello', nil].map(&:to_i).compact
=> [1, 2, 0, 0, 0]
irb(main):007:0> ['1', '2', '', 'hello', nil].filter_map(&:to_i)
=> [1, 2, 0, 0, 0]
```
Doesn't look like a change of behavior, so seems fine - just wanted to check!

Reply:
Oh, yeah, to_i shouldn't return false, so the behavior change between map+compact and filter map I noted in the comment is probably fine.